code/obj/item/clothing/chameleon_js.dm

```diff
@@ -1250,23 +1250,23 @@
 		scramble_prints = 0
 
 	fingerless
-		desc = "These gloves lack fingers."
-		name = "Fingerless Gloves"
+		desc = "These gloves lack fingers. Good for a space biker look, but not so good for concealing your fingerprints."
+		name = "fingerless gloves"
 		icon_state = "fgloves"
 		item_state = "finger-"
 		hide_prints = 0
 		scramble_prints = 0
 
 	latex
-		name = "Latex Gloves"
+		name = "latex gloves"
 		icon_state = "latex"
 		item_state = "lgloves"
-		desc = "Thin gloves that offer minimal protection."
+		desc = "Thin, disposal medical gloves used to help prevent the spread of germs."
 		scramble_prints = 1
 
 	boxing
-		name = "Boxing Gloves"
-		desc = "These gloves are for competitive boxing."
+		name = "boxing gloves"
+		desc = "Big soft gloves used in competitive boxing. Specifically designed to stop your fists from inflicting permanent injuries, so your punches will knock people down without hurting them as much."
```

Comment:
a bit misleading here - they don't actually reduce damage at all, they just trade random stamina-criticals for bonus average stamina damage

Reply:
I was trying to mull over how to portray that, but I was drawing a blank. Any ideas?

Reply:
maybe something like `gives your punches a bit more weight at the cost of precision`?

'Weight' isn't perfect but I can't think of a word to portray 'knocking people over more without hurting them more'

Likewise 'precision' doesn't exactly convey 'Criticals', but thinking in an RP sense critting someone is probably just punching them in the kidney or the jaw or whatever other sensitive bits

Reply:
sure, I vibe with that